src/vm/ClrEtwAll.man

```diff
@@ -364,16 +366,27 @@
                     <task name="DebugExceptionProcessing" symbol="CLR_EXCEPTION_PROCESSING_TASK"
                           value="26" eventGUID="{C4412198-EF03-47F1-9BD1-11C6637A2062}"
                           message="$(string.RuntimePublisher.DebugExceptionProcessingTaskMessage)">
-                    <opcodes>
-                    </opcodes>
-                  </task>
+                        <opcodes>
+                        </opcodes>
+                    </task>
                     <task name="CodeSymbols" symbol="CLR_CODE_SYMBOLS_TASK"
                           value="30" eventGUID="{53aedf69-2049-4f7d-9345-d3018b5c4d80}"
                           message="$(string.RuntimePublisher.CodeSymbolsTaskMessage)">
-                    <opcodes>
-                    </opcodes>
-                  </task>
-                <!--Next available ID is 31-->
+                        <opcodes>
+                        </opcodes>
+                    </task>
+                    <task name="TieredCompilation" symbol="CLR_TIERED_COMPILATION_TASK"
+                          value="31" eventGUID="{A77F474D-9D0D-4311-B98E-CFBCF84B9E0F}"
+                          message="$(string.RuntimePublisher.TieredCompilationTaskMessage)">
+                        <opcodes>
+                            <opcode name="Settings" message="$(string.RuntimePublisher.TieredCompilationSettingsOpcodeMessage)" symbol="CLR_TIERED_COMPILATION_SETTINGS_OPCODE" value="11"/>
+                            <opcode name="Pause" message="$(string.RuntimePublisher.TieredCompilationPauseOpcodeMessage)" symbol="CLR_TIERED_COMPILATION_PAUSE_OPCODE" value="12"/>
+                            <opcode name="Resume" message="$(string.RuntimePublisher.TieredCompilationResumeOpcodeMessage)" symbol="CLR_TIERED_COMPILATION_RESUME_OPCODE" value="13"/>
+                            <opcode name="BackgroundJitStart" message="$(string.RuntimePublisher.TieredCompilationBackgroundJitStartOpcodeMessage)" symbol="CLR_TIERED_COMPILATION_BACKGROUND_JIT_START_OPCODE" value="14"/>
```

Comment:
Rather than using BackgroundJitStart and BackgroundJitStop opcodes, I'd recommend just using win:start and win:stop.  Tools will know how to handle these automatically.  As an example, PerfView will automatically calculate the elapsed time between these events and show it to you in the Events view.